src/main.cc

```diff
@@ -1690,6 +1794,60 @@ Parse(TRITONSERVER_ServerOptions** server_options, int argc, char** argv)
   sagemaker_safe_range_ = sagemaker_safe_range;
 #endif  // TRITON_ENABLE_SAGEMAKER
 
+#if defined(TRITON_ENABLE_ADSBRAIN)
+  adsbrain_port_ = adsbrain_port;
+  adsbrain_thread_cnt_ = adsbrain_thread_cnt;
+
+  // entrypoint
+  char* v = getenv(adsbrain_entrypoint_env.c_str());
```

Comment:
A doc about how to run triton-adsbrain-server will be helpful.

Reply:
will update our docs in sharepoint.

Reply:
added doc.